include/miopen/miopen.h

```diff
@@ -2570,6 +2570,91 @@ MIOPEN_EXPORT miopenStatus_t miopenLayerNormForward(miopenHandle_t handle,
 // CLOSEOUT LAYERNORM DOXYGEN GROUP
 #endif
 
+#ifdef MIOPEN_BETA_API
+// MarginRankingLoss APIs
+/** @addtogroup marginrankingloss
+ *
+ *  @{
+ */
+
+typedef enum
+{
+    MIOPEN_MARGINRANKINGLOSS_REDUCTION_NONE = 0, /*!< output tensor elements are not reduced */
```

Comment:
I'm starting to think that we should unify these reduction mode enums..
There are a few thing like it, so I'll organize and share them, alongside with tensor_view macros,  before opening PR to ROCm/MIOpen. Let's leave it for now.

Reply:
Got it, I will syn-up with our team about `enum` unification.

Reply:
I think defining miopenLossReductionMode_t is fine for many loss operations. Please check [my PR implementation](https://github.com/moreh-dev/MIOpen/pull/33/files)